images/public-log-vendor-matcher/pg-init.d/lib-00-prepare.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+set -xeo pipefail
+
+PIPELINE_DATE="${PIPELINE_DATE:-$(date +%Y%m%d)}"
+export PIPELINE_DATE
+
+cat << EOF > "$HOME/.bigqueryrc"
+credential_file = ${GOOGLE_APPLICATION_CREDENTIALS}
+project_id = ${GCP_PROJECT}
+EOF
+
+gcloud config set project "${GCP_PROJECT}"
+
+## This is just to continue testing wile I wait for permissions for the service account
+## Use the activate-service-account live once it has permissions
+## The container is being run it so it should let me manually do the auth
+# gcloud auth login
+gcloud auth activate-service-account "${GCP_SERVICEACCOUNT}" --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
```

Comment:
You shouldn't need to specify the name of the service account, e.g. https://github.com/kubernetes/test-infra/blob/3896f0bc47a2fa00e20f7c6ef3640681314a78c6/metrics/bigquery.py#L154-L157
```suggestion
gcloud auth activate-service-account  --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
```

Reply:
Removed in ad9fe407